geode-core/src/test/java/org/apache/geode/cache/client/internal/PutOpJUnitTest.java

```diff
@@ -16,19 +16,38 @@
 
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
+import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mockito;
 
 import org.apache.geode.cache.Operation;
 import org.apache.geode.internal.cache.EntryEventImpl;
 import org.apache.geode.internal.cache.EventID;
+import org.apache.geode.internal.cache.RegionEntry;
+import org.apache.geode.internal.cache.versions.VersionStamp;
+import org.apache.geode.internal.cache.versions.VersionTag;
 
 public class PutOpJUnitTest {
+  private final EntryEventImpl event = mock(EntryEventImpl.class);
+  private final VersionTag versionTag = mock(VersionTag.class);
+  private final RegionEntry entry = mock(RegionEntry.class);
+  private final VersionStamp versionStamp = mock(VersionStamp.class);
+  private final Connection connection = mock(Connection.class);
+
+  @Before
+  public void setup() {
+    when(event.getEventId()).thenReturn(new EventID());
+    when(entry.getVersionStamp()).thenReturn(versionStamp);
+  }
 
   private EntryEventImpl getEntryEvent() {
-    EntryEventImpl entryEvent = Mockito.mock(EntryEventImpl.class);
-    Mockito.when(entryEvent.getEventId()).thenReturn(new EventID());
+    EntryEventImpl entryEvent = mock(EntryEventImpl.class);
+    when(entryEvent.getEventId()).thenReturn(new EventID());
     return entryEvent;
   }
```

Comment:
I think this method can be entirely removed now that there is an `event` field in the test, and calls to it can be replaced with usages of the field.

Reply:
Done